app/models/poll/question/answer.rb

```diff
@@ -1,6 +1,11 @@
 class Poll::Question::Answer < ActiveRecord::Base
   include Galleryable
   include Documentable
+
+  translates :title,       touch: true
+  translates :description, touch: true
+  globalize_accessors locales: [:en, :es, :fr, :nl, :pt_br]
```

Comment:
Is there any reason to specify the locales explicitly here?

Reply:
Not anymore https://github.com/consul/consul/pull/2914/commits/46030c72e829006da92bb02892eac893c1823893